label_options/caebike-label-options.json

```diff
@@ -0,0 +1,85 @@
+{
+  "MODE": [
+    {"value":"walk", "baseMode":"WALKING", "met": {"ALL": {"range": [0, -1], "mets": 3}}, "kgCo2PerKm": 0},
+    {"value":"auto_driver", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.22031},
+    {"value":"auto_passenger", "baseMode":"CAR", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.110155},
+    {"value":"e-bike", "baseMode":"E_BIKE", "met": {"ALL": {"range": [0, -1], "mets": 5}}, "kgCo2PerKm": 0.00728},
+    {"value":"bike", "baseMode":"BICYCLING", "met": {"ALL": {"range": [0, -1], "mets": 6}}, "kgCo2PerKm": 0},
+    {"value":"taxi", "baseMode":"TAXI", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.30741},
+    {"value":"bike_scooter_share", "baseMode":"BICYCLING_SCOOTER_SHARE", "met": {"ALL": {"range": [0, -1], "mets": 3.5}}, "kgCo2PerKm": 0.00447},
+    {"value":"bus_train_shuttle", "baseMode":"PUBLIC_TRANSIT", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.179033},
```

Comment:
"PUBLIC_TRANSIT" is also not a base mode we support, I'd suggest picking one from {BUS, TRAIN, TRAM, LIGHT_RAIL, SUBWAY, TROLLEYBUS} - maybe whichever is most present in the area of interest?